### PR TITLE
Neato config flow

### PR DESCRIPTION
**Pull request by Santobert**
**Description:**
Update the documentation according to the recently changed configuration process of neato integration.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#26579

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/neato.markdown [hunk 1]
```diff
@@ -15,7 +15,18 @@ redirect_from:
...
 The `neato` integration allows you to control your [Neato Botvac Connected Robots](https://www.neatorobotics.com/robot-vacuum/botvac-connected-series/).
 
-To enable `neato` in your installation, add the following to your `configuration.yaml` file:
+To activate `neato` in your installation, you can set it up from the integration screen or add it to your `configuration.yaml` file.
+
+## Setup the integration via the integrations screen
+
+Menu: *Configuration* -> *Integrations*
+
+Press on **Neato** and configure the integration. You will need to enter your username and password and whether you are using a Neato or Vorwerk device.
```
Comment (dshokouhi) [inline source/_components/neato.markdown]:
```suggestion
Search for or select **Neato** from the list and configure the integration. You will need to enter your username and password and whether you are using a Neato or Vorwerk device.
```

**Review by dshokouhi — COMMENTED** [review]
Just a small change for readability

File: source/_components/neato.markdown (hunk 1 above)
Comment (Santobert) [reply source/_components/neato.markdown]: Done, thank you

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by Santobert** [comment]
@frenck I just had some problems with merging... no content was changed

**Review by frenck — DISMISSED** [review]

**Review by MartinHjelmare — DISMISSED** [review]

**Comment by klaasnicolaas** [comment]
missing the `ha_config_flow: true` at the top 😄

**Comment by Santobert** [comment]
> missing the `ha_config_flow: true` at the top smile

Added that. Is it correct?

**Review by frenck — APPROVED** [review]
Thanks, @Santobert! That looks good now! 👍

**Pull request merged**